torch/_export/serde/serialize.py

```diff
@@ -1087,6 +1153,8 @@ def serialize_argument_spec(self, x: ep.ArgumentSpec) -> Argument:
             return Argument.create(as_tensor=TensorArgument(name=x.name))
         elif isinstance(x, ep.SymIntArgument):
             return Argument.create(as_sym_int=SymIntArgument.create(as_name=x.name))
+        elif isinstance(x, ep.SymFloatArgument):
```

Comment:
I guess you need to add SymFloatArgument [here](https://github.com/pytorch/pytorch/blob/main/torch/export/graph_signature.py) too, but if you want you can ignore serializing SymFloats as arguments in this PR?

Reply:
I don't need it currently for my case but it is done now.